### ISSUE TITLE
Exception while saving- return from manned space station via gemini capsule.

### ISSUE THREAD

**Issue by byt3c** [jrossignol/ContractConfigurator#731]
Hello - there is an apparent issue with contract configurator while returning from manned space station on gemini capsule.

![image](https://github.com/user-attachments/assets/b10e2585-3d76-428a-93be-7d7507e2a469)

Exception occured while saving ScenarioModule 'ContractVesselTracker':
System.NullReferenceException: Object reference not set to an instance of an object
  at ContractConfigurator.Extensions+<>c__DisplayClass5_1.<GetHashes>b__1 (ProtoPartSnapshot childPart) [0x00000] in <0ac19348e13c43e280580b62f08ea2eb>:0 
  at System.Linq.Enumerable+WhereListIterator`1[TSource].MoveNext () [0x00037] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Boolean& found) [0x00045] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00000] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at ContractConfigurator.Extensions+<GetHashes>d__5.MoveNext () [0x00320] in <0ac19348e13c43e280580b62f08ea2eb>:0 
  at System.Linq.Enumerable.Contains[TSource] (System.Collections.Generic.IEnumerable`1[T] source, TSource value, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x00036] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Enumerable.Contains[TSource] (System.Collections.Generic.IEnumerable`1[T] source, TSource value) [0x0000a] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at ContractConfigurator.ContractVesselTracker.OnSave (ConfigNode node) [0x00085] in <0ac19348e13c43e280580b62f08ea2eb>:0

**Comment by dollythecamus** [comment]
hi, I'm useless but I literally just had the same exact issue, returning kerbals from my space station. have you found a fix?

**Comment by byt3c** [comment]
Nope, but it happened only at third return. Maybe because those kerbals were detatching from space station that was running the experiment. ? Or one of them is a scientist ? I have no idea. Two new other replacement kerbals are still there.

**Comment by byt3c** [comment]
![image](https://github.com/user-attachments/assets/9d6a7714-70ac-442b-9539-2798765c17cd)
after loading previous save i'm still getting this even though i tried exiting scene and landing after loading station separately after docking. I haven;t tried loading lander craft before landing.

**Comment by dollythecamus** [comment]
since I commented, I reloaded to a save before the crew return with no issues yet, I still didn't return the crew though
